include/NovelRT/Atom.h

```diff
@@ -71,29 +72,33 @@ namespace NovelRT
             return _value;
         }
 
-        // TODO: These should be internal to NovelRT
-
-        static Atom GetNextEventHandlerId() noexcept;
-
-        static Atom GetNextFontSetId() noexcept;
-
-        static Atom GetNextTextureId() noexcept;
-
-        static Atom GetNextComponentTypeId() noexcept;
-
-        static Atom GetNextEntityId() noexcept;
-
-        static Atom GetNextSystemId() noexcept;
+        static Atom GetNextEcsPrimitiveInfoConfigurationId() noexcept;
+    };
 
-        static Atom GetNextEcsTextureId() noexcept;
+    class AtomFactory
```

Comment:
Why is this not included in its own header file? Seems partially inconsistent with our standards but I could be confusing the old way vs new way

Reply:
because I do sometimes group types that are used basically all the time together occasionally this isn't convention or anything